dabest/_classes.py

```diff
@@ -470,6 +488,13 @@ def is_paired(self):
         """
         return self.__is_paired
 
+    @property
+    def is_preceding(self):
```

Comment:
Please change `preceding` to `repeated_measures` and `is_preceding` to ` repeated_measures`